### ISSUE TITLE
On https://openshift.io, creating Che workspace results in workspace missing "run" command

### ISSUE THREAD

**Issue by ldimaggi** [openshiftio/openshift.io#1873]
Steps to recreate:
* Create new space
* Create new quickstart
* Create new Che workspace for quickstart
* Observe that "run" command is missing in workspace:

![screenshot from 2018-01-12 08-17-33](https://user-images.githubusercontent.com/642621/34876820-4b922a82-f771-11e7-94ab-830be9ae597f.png)
![screenshot from 2018-01-12 08-17-20](https://user-images.githubusercontent.com/642621/34876821-4bb26af4-f771-11e7-9d72-d8a7e40fcefb.png)

**Comment by rhopp** [comment]
How are you creating the workspace... I guess from openshift.io codebases page, right?
Is it reproducible with different projects? Or is it reproducible with some specific project? What project?

**Comment by rhopp** [comment]
I was able to reproduce this on prod with vertx http booster project.

**Comment by ldimaggi** [comment]
Yes - just create a new quickstart.

**Comment by ldimaggi** [comment]
Haven't we seen this problem in the past too?

**Comment by eivantsov** [comment]
@ldimaggi yes we have. I think the wrong stack is being pulled

**Comment by rhopp** [comment]
It seems like java-centos stack is being used instead of vert.x
```
{"time":"2018-01-12T13:30:35.574+00:00","msg":"Stack: java-centos","logger":"io.fabric8.che.starter.client.WorkspaceClient","thread":"qtp1308109015-44737","level":"INFO","HOSTNAME":"che-starter-44-0mm55","req_id":"d1d44a04-2d79-40ff-a80c-97ca78808abf","identity_id":"25235c85-b434-4572-872d-b7c6a6d7d992"}
 ```

**Comment by ibuziuk** [comment]
@joshuawilson @corinnekrych could you please check if it is not a regression of https://github.com/openshiftio/openshift.io/issues/1137

**Comment by joshuawilson** [comment]
@ibuziuk I'm not sure if it is a regression or just the same error introduced again. Isn't it a Che issue though, or do you think this is set in the wizard?

**Comment by joshuawilson** [comment]
@gastaldi looks like this is for you.

**Comment by ldimaggi** [comment]
Can we get this issue addressed? It makes it difficult for user (and automated tests) to run an app in Che - thx!

**Comment by gastaldi** [comment]
@ldimaggi I'll look into it

**Comment by gastaldi** [comment]
Pushed fix to staging. @ldimaggi See if it works, then I'll update production

**Comment by gastaldi** [comment]
After https://ci.centos.org/view/Devtools/job/devtools-launcher-backend-generator-build-master/80/console is done, of course

**Comment by ldimaggi** [comment]
Have tried this multiple times on prod-preview - and could never get past this point:

![screenshot from 2018-01-18 08-28-49](https://user-images.githubusercontent.com/642621/35100408-b9fe85ee-fc29-11e7-82dd-a0cadb3833e1.png)

**Comment by gastaldi** [comment]
It appears to be something wrong with your user. It works for me: 

![screenshot from 2018-01-18 11-39-28](https://user-images.githubusercontent.com/54133/35100810-4774223e-fc44-11e7-86d0-3b7428107d1b.png)

**Comment by ldimaggi** [comment]
Have tried multiple times - will try with a diff account.

**Comment by ldimaggi** [comment]
OK - it seems fine with a diff account:

![screenshot from 2018-01-18 08-55-16](https://user-images.githubusercontent.com/642621/35101475-b0dd5bf8-fc2d-11e7-9f6c-29c27d4a81a6.png)

But - how can we debug this? Why are some accounts blocked by this - while others are not?

**Comment by gastaldi** [comment]
I have no idea. Maybe it would be nice to have an issue assigned to the Che team to investigate this

**Comment by ldimaggi** [comment]
Issue found/encountered by EE tests.
